rustls/src/msgs/message.rs

```diff
@@ -72,51 +73,30 @@ impl MessagePayload {
 /// A TLS frame, named TLSPlaintext in the standard.
 /// This type owns all memory for its interior parts.
 #[derive(Debug)]
-pub struct Message {
+pub struct RawMessage {
     pub typ: ContentType,
     pub version: ProtocolVersion,
-    pub payload: MessagePayload,
+    pub payload: Payload,
```

Comment:
I've moved the constants from `Message` to `OpaqueMessage`. They're only used in the context of the deframer and they might not really apply to `Message` payloads anyway (since those might have been joined).